Sources/AccioKit/Commands/Protocols/DependencyInstaller.swift

```diff
@@ -52,7 +52,8 @@ extension DependencyInstaller {
         workingDirectory: String = GlobalOptions.workingDirectory.value ?? FileManager.default.currentDirectoryPath,
         manifest: Manifest,
         dependencyGraph: DependencyGraph,
-        sharedCachePath: String?
+        sharedCachePath: String?,
+        toolchain: String? = nil
```

Comment:
Why is this nil here?

Reply:
It is nil to fallback to the default behaviour as before to avoid breaking changes.